src/tests/FunctionalTests/Android/Device_Emulator/AOT_PROFILED/Android.Device_Emulator.Aot_Profiled.Test.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <MonoForceInterpreter>false</MonoForceInterpreter>
+    <RunAOTCompilation>true</RunAOTCompilation>
+    <AOTWithLibraryFiles>true</AOTWithLibraryFiles>
+    <TestRuntime>true</TestRuntime>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <MainLibraryFileName>Android.Device_Emulator.Aot_Profiled.Test.dll</MainLibraryFileName>
+    <ExpectedExitCode>42</ExpectedExitCode>
+    <EnableAggressiveTrimming>true</EnableAggressiveTrimming>
+  </PropertyGroup>
+
+  <!-- Tracing -->
+  <PropertyGroup>
+    <!-- <RuntimeComponents>diagnostics_tracing</RuntimeComponents> -->
+    <!-- <DiagnosticPorts>127.0.0.1:9000,suspend</DiagnosticPorts> -->
+  </PropertyGroup>
+
+  <!-- Profiled AOT with .nettrace -->
+  <PropertyGroup>
+    <NetTraceFilePath>$(MSBuildThisFileDirectory)\adeapt.nettrace</NetTraceFilePath>
+  </PropertyGroup>
+
+  <!-- Profiled AOT with .mibc-->
+  <PropertyGroup>
+    <ProfiledAOTProfilePaths>$(MSBuildThisFileDirectory)\adeapt.mibc</ProfiledAOTProfilePaths>
```

Comment:
you might want to call this just test.mibc and test.nettrace so it is clearer that these are test assets.